projects/schematics/src/shared/utils/lib-utils.ts

```diff
@@ -306,6 +280,8 @@ export function addLibraryFeature<T extends LibraryOptions>(
       config.dependencyManagement
         ? installRequiredSpartacusFeatures(config.dependencyManagement, options)
         : noop(),
+
+      orderInstalledFeatures(options),
```

Comment:
Not sure if it makes sense to do it now, but instead of ordering at every spartacus lib installation, maybe we can just do it at the end.

Can't find where, but I'm assuming there's a tree of things, where we know how many features are going to be installed beforehand, so we maybe only run after the last added installation to slightly improve performance.